src/Swarm/Game/Step.hs

```diff
@@ -1230,6 +1316,34 @@ execConst c vs s k = do
       _ -> badConst
     Grab -> doGrab Grab'
     Harvest -> doGrab Harvest'
+    Ignite -> case vs of
```

Comment:
Might there be a benefit to doing something like `use "torch"` instead of adding yet another command?  I am not sure how it would work exactly, just wondering.

Reply:
So far, the `use` command exclusively applies a **recipe** to an **entity** to produce another **entity**.  `ignite` is closer to the `harvest` command, in that the application of the command produces a **robot**.  I think it would be weird to special-case `use` for this purpose.

Reply:
But perhaps a `"torch"` could provide the `Capability` that enables the `ignite` command.

Reply:
If we end up with a lot of commands like ignite/harvest/… that have effects, then we could generalise `use` and drop the commands.

It might be nice to have #455 for that.